mindsdb/integrations/handlers/whatsapp_handler/whatsapp_handler.py

```diff
@@ -367,8 +369,8 @@ def send_message(self, params, ret_as_dict=False) -> Response:
 
         except Exception as e:
             # Log the exception for debugging purposes
-            log.logger.error(f"Error sending message: {str(e)}")
-            raise Exception(f"Error posting message to the user '{params['to_number']}': {e.response['error']}")
+            logger.exception(f"Error sending message: {str(e)}")
```

Comment:
Should we log error here too?

Reply:
Oh yeah, thanks for pointing it out. Just updated the code.